packages/embark-utils/src/web3Utils.ts

```diff
@@ -0,0 +1,51 @@
+import { __ } from "i18n";
+import { balanceRegex } from "./constants";
+const Web3EthAbi = require("web3-eth-abi");
+const Web3 = require("web3");
```

Comment:
I would make it so that this file is the only utils file that requires Web3, and when you want to use those functions, you do `import Web3Utils from 'embark-utils/web3Utils'`, so that way you only require web3 when you need it. Does it make sense? Also, is it doable?